air/src/aux_table/bitwise_pow2/bitwise/mod.rs

```diff
@@ -151,12 +149,13 @@ fn enforce_input_decomposition<E: FieldElement>(
     constraint_offset
 }
 
-/// Enforces correct output aggregation for the operation. This requires the following 2 constraints
+/// Enforces correct output aggregation for the operation. This requires the following 3 constraints
 /// for each operation:
-/// - In the first row, the output value must equal the aggregated result of the operation applied
-///   to the row.
-/// - For every row except the last, the output value in the next row must equal the output in the
-///   current row plus the aggregated result of the specified operation applied to the next row.
+/// - In the first row, the aggregated output value of the previous row must be zero.
+/// - For every row except the last, the aggregated output value must equal the previous aggregated
+///   output value in the next row.
+/// - For every row, the output value must equal 16 times the previous output plus the aggregated
+///   result of the specified operation.
```

Comment:
nit: Sorry I missed this last time, but I think the output aggregation constraint descriptions used in the `pow2` module are a bit clearer (they had more eyes on them). Since this is doing the same thing maybe we should just reuse those descriptions here:

```
/// - In the first row, `output_prev` should be set to 0.
/// - For all the rows except the last one, the next value of `output_prev` should be the same as
///   the current value of `output`.
/// - For all rows except the last one, the current output value (`output`) should equal the output value
///   copied from the previous row (`output_prev`) plus the aggregation of current row's decomposed power.
```